lib/helpers/campaign.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const dateFns = require('date-fns');
```

Comment:
Just wondering why the choice of `date-fns` over the seemingly more popular `moment` [module](https://www.npmjs.com/package/moment).

Reply:
[Code was stolen from Phoenix Next](https://github.com/DoSomething/phoenix-next/blob/e6711c9330c026e30321eaa4980a093b83df9cf7/resources/assets/helpers/index.js#L331) 🕵️

Reply:
@rapala61 the reason specifically is that `date-fns` has a significantly smaller footprint on the code base.